frontend/src/scenes/App.tsx

```diff
@@ -60,7 +61,22 @@ export const appLogic = kea<appLogicType<PreflightStatus>>({
 
 export function App(): JSX.Element | null {
     const { showApp, showingDelayedSpinner } = useValues(appLogic)
-    return showApp ? <AppScene /> : showingDelayedSpinner ? <SceneLoading /> : null
+    const { user } = useValues(userLogic)
+
+    return showApp ? (
```

Comment:
Nitpick: This ternary is a bit much now IMO. Worth extracting the first case like this?

```ts
if (showApp) {
  return (
    <>            
      {user ? <Models /> : null}
      <AppScene />
    </>
  )
}